packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -290,10 +291,12 @@ export const prepareAuctionBook = (baggage, zcf) => {
               : AmountMath.makeEmptyFromAmount(want);
 
           const stillWant = AmountMath.subtract(want, collateralSold);
+          trace('acceptScaledBidOffer', { stillWant });
```

Comment:
Any particular reason we're adding tracing to `auctionBook` in order to add `tryExit` to the smart wallet?